src/OSPSuite.UI/Views/Importer/ColumnMappingView.cs

```diff
@@ -344,15 +346,24 @@ public string Format(DataFormatParameter model)
 
          if (model is MappingDataFormatParameter mapping)
          {
-            var str = $"Units: {mapping.MappedColumn.Unit.SelectedUnit}";
+            var str = "";
+
+            if (!mapping.MappedColumn.Unit.ColumnName.IsNullOrEmpty())
+            {
```

Comment:
same stuff. Ensure the same styling in the same function. Otherwise the code is not cohesive

Reply:
done